code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,53 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+/datum/status_effect/pranked
+	id = "pranked"
+	duration = 20 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = null
+	///increments every time the status effect is refreshed. Higher counter == more powerful clown combos.
+	var/prank_counter = 1
+
+/datum/status_effect/pranked/refresh()
+	. = ..()
+	prank_counter++
+/datum/status_effect/slapped_silly
```

Comment:
put a space between the end of `refresh()` and the start of `/datum/status_effect/slapped_silly`